Scripts/Interfaces/Entity/IMainMenu.cs

```diff
@@ -23,7 +23,12 @@ public interface IMainMenu : IMenu, IUsesMenuActions, ISelectTool
 		/// <summary>
 		/// The types which provide a settings menu
 		/// </summary>
-		Dictionary<Type, ISettingsMenuProvider> settingsMenuProviders { set; }
+		Dictionary<KeyValuePair<Type, Transform>, ISettingsMenuProvider> settingsMenuProviders { set; }
```

Comment:
Menu Item provider is for adding single items like this toggle group to the settings menu, as opposed to entire sub-faces